PGM/src/main/java/tc/oc/pgm/flag/Flag.java

```diff
@@ -483,9 +484,20 @@ public void playStatusSound(BukkitSound ownerSound, BukkitSound otherSound) {
         }
     }
 
+    private boolean isFlagItem(ItemStack item) {
+        if (item.getItemMeta() instanceof BannerMeta) {
+            ItemStack banner = new ItemStack(item);
+            BannerMeta bannerMeta = (BannerMeta) banner.getItemMeta();
+            DyeColor dyeColor = DyeColor.getByDyeData(banner.getData().getData());
+            bannerMeta.setBaseColor(dyeColor);
+            banner.setItemMeta(bannerMeta);
```

Comment:
Why is the banner data being reset? It seems to get the banner's dye color and just set it again.